nautobot/core/settings.py

```diff
@@ -586,15 +594,15 @@
     ],
     "BANNER_BOTTOM": [
         "",
-        "Custom HTML to display in a banner at the bottom of all pages.",
+        "Custom Markdown or limited HTML to display in a banner at the bottom of all pages.",
```

Comment:
Would it make more sense to write "`pure HTML without css or javascript`" in place of `limited HTML` to be more clear??

Reply:
The docs describe exactly which subset of HTML is permitted, but that'd be too verbose to spell out here I think.